.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "@logux/eslint-config/ts",
+  "plugins": ["prettier"],
```

Comment:
We do not use prettier eslint in other projects

Reply:
By the way, why it is so? =)

Do you mean that `prettier` must be launched separately from `eslint` every time ?

Reply:
Prettier is 40 MB (!) dependency.

I prefer to use Prettier plugin in text editor and avoid having the copy in `node_modules`.